editor.planx.uk/src/pages/FlowEditor/components/Flow/components/Tag.tsx

```diff
@@ -1,35 +1,60 @@
 import Box from "@mui/material/Box";
-import { visuallyHidden } from "@mui/utils";
+import { PaletteOptions, useTheme } from "@mui/material/styles";
 import { NodeTag } from "@opensystemslab/planx-core/types";
+import { useStore } from "pages/FlowEditor/lib/store";
 import React from "react";
+import { getContrastTextColor } from "styleUtils";
 import { FONT_WEIGHT_SEMI_BOLD } from "theme";
 
 export const TAG_DISPLAY_VALUES: Record<
```

Comment:
Not sure if I have the right colour scheme or setup here @ianjon3s , should be be a quick and easy fix though!